ergoCubSN000/head.xml

```diff
@@ -19,10 +19,6 @@
     <xi:include href="wrappers/inertials/head-inertials_wrapper.xml" />
     <xi:include href="hardware/inertials/head-inertial.xml" />
 
-   <!-- Head sensors-->
-    <xi:include href="sensors/realsense.xml" />
-    <xi:include href="sensors/rplidar.xml" />
-
```

Comment:
Hi @AntonioConsilvio 

Could you clarify the rationale for this removal?
Thanks!

Reply:
Yes @pattacini . This `head.xml` file activates the neck motors and realsense. 

But it was unusable on both the Xavier and the torso CPUs, because if you start a `yarprobotinterface --config head.xml` on the torso CPU, it cannot start because the realsense is connected to the Xavier. 

At the same time, it cannot start on the Xavier because the motor control boards are connected to the torso CPU.

So this file was unusable.

I therefore removed the `head sensors` part so that the file could be started on the torso, and the realsense can be started separately on the Xavier with `realsense2.xml` or `sensors.xml`.

cc @mfussi66

Reply:
Thanks for the explanation!